velox/exec/tests/utils/HiveConnectorTestBase.cpp

```diff
@@ -61,35 +61,38 @@ void HiveConnectorTestBase::resetHiveConnector(
 
 void HiveConnectorTestBase::writeToFile(
     const std::string& filePath,
-    RowVectorPtr vector) {
-  writeToFile(filePath, std::vector{vector});
+    RowVectorPtr vector,
+    dwio::common::FileFormat fileFormat) {
+  writeToFile(filePath, std::vector{vector}, {}, fileFormat);
 }
 
 void HiveConnectorTestBase::writeToFile(
     const std::string& filePath,
     const std::vector<RowVectorPtr>& vectors,
-    std::shared_ptr<dwrf::Config> config) {
-  writeToFile(filePath, vectors, std::move(config), vectors[0]->type());
+    dwio::common::WriterOptions options,
+    dwio::common::FileFormat fileFormat) {
+  writeToFile(filePath, vectors, options, vectors[0]->type(), fileFormat);
 }
 
 void HiveConnectorTestBase::writeToFile(
     const std::string& filePath,
     const std::vector<RowVectorPtr>& vectors,
-    std::shared_ptr<dwrf::Config> config,
-    const TypePtr& schema) {
-  velox::dwrf::WriterOptions options;
-  options.config = config;
-  options.schema = schema;
+    dwio::common::WriterOptions options,
+    const TypePtr& schema,
```

Comment:
See if we can remove this overload if it's only used by the above one.

Reply:
The test TableScanTest::readFlatMapAsStruct needs the schema parameter in TableScanTest.cpp. `writeToFile(file->getPath(), {vector}, options, writeSchema);` so I can't remove this overload